src/TeamContributionCalendar/TeamContributionCalendar.js

```diff
@@ -45,6 +45,12 @@ export default class TeamContributionCalendar {
       throw new Error(defaultUserData.errorMessage);
     }
 
+    if (!this.users.gitHub.length && !this.users.gitLab.length) {
```

Comment:
https://github.com/c-hive/team-contribution-calendar/blob/a63c49e5128c416f37711ab06da958b52fcf49d2/src/TeamContributionCalendar/TeamContributionCalendar.js#L170-L189

Just realized that the flag is set to false right after the first GitHub/GitLab has been merged. In this case, I'd bind the indicator's state to a conditional check.

Reply:
I don't get this. Does the PR solve the issue currently?

Reply:
Yep.